tools/deployment/package-helm/templates/results-cache-indices-creator-job.yaml

```diff
@@ -39,6 +35,8 @@ spec:
           command: [
             "python3", "-u",
             "-m", "clp_py_utils.initialize-results-cache",
-            "--uri", "mongodb://{{ include "clp.fullname" . }}-results-cache:27017/{{ .Values.clpConfig.results_cache.db_name }}",
-            "--stream-collection", {{ .Values.clpConfig.results_cache.stream_collection_name | quote }}
+            "--uri", "mongodb://{{ include "clp.fullname" . }}-results-cache:27017/{{
```

Comment:
these lines are unrelated to the main purpose of the PR. they're only updated to avoid code style violations (exceeding max 100 chars per line)